packages/medusa/src/repositories/publishable-api-key-sales-channel.ts

```diff
@@ -59,20 +60,23 @@ export const PublishableApiKeySalesChannelRepository = dataSource
      *
      * @param publishableApiKeyId - publishable key id
      * @param salesChannelIds - an array of SC ids
+     * @param isMedusaV2Enabled - are Medusa modules enabled
      */
     async addSalesChannels(
       publishableApiKeyId: string,
-      salesChannelIds: string[]
+      salesChannelIds: string[],
+      isMedusaV2Enabled?: boolean
```

Comment:
comment: Don't think feature flags should be part of the public-facing service method. Also, what is this used for?

Reply:
forgot to remove the argument during refactor, it's not needed anymore